app/services/rate_associator.rb

```diff
@@ -5,7 +5,9 @@
 class RateAssociator
   def initialize(child)
     @child = child
-    @region = Illinois::RegionFinder.new(business: child.business).call
+    child_business = @child.child_businesses.find_by(currently_active: true)
+    active_business = Business.find(child_business.business_id)
```

Comment:
Why these two lines are needed(8 and 9)?

Wouldn't this be enough? 
active_business = @child.child_businesses.find_by(currently_active: true)